src/modules/users/components/TokenActivation/TokenActivationContent/TokensTab.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { BigNumber } from 'ethers/utils';
+
+import Icon from '~core/Icon';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import Numeral from '~core/Numeral';
+
+import { UserToken } from '~data/generated';
+import { formatTokenValue } from '~utils/numbers';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './TokenActivationContent.css';
+import ChangeTokenStateForm from './ChangeTokenStateForm';
+import TokenTooltip from './TokenTooltip';
+
+const MSG = defineMessages({
+  active: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.active',
+    defaultMessage: 'Active',
+  },
+  inactive: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.inactive',
+    defaultMessage: 'Inactive',
+  },
+  activeTokensTooltip: {
+    id:
+      // eslint-disable-next-line max-len
+      'users.TokenActivation.TokenActivationContent.TokensTab.activeTokensTooltip',
+    defaultMessage: `Tokens are “Active” when they’ve been deposited to a
+     contract which lets them get ‘locked’ when you need to stake,
+     or claim a share of Rewards. You can withdraw tokens back
+     to your wallet any time, you just need to clear any locks first.`,
+  },
+  inactiveTokensTooltip: {
+    id:
+      // eslint-disable-next-line max-len
+      'users.TokenActivation.TokenActivationContent.TokensTab.inactiveTokensTooltip',
+    defaultMessage: `Inactive tokens are contained in your own wallet.
+     You need to “Activate” them to stake, or be eligible to receive Rewards.`,
+  },
+  lockedTokensTooltip: {
+    id:
+      // eslint-disable-next-line max-len
+      'users.TokenActivation.TokenActivationContent.TokensTab.lockedTokensTooltip',
+    defaultMessage: `You have unclaimed transactions which must be claimed
+     before these tokens can be withdrawn.`,
+  },
+});
+
+export interface TokensTabProps {
+  activeTokens: BigNumber;
+  inactiveTokens: BigNumber;
+  totalTokens: BigNumber;
+  lockedTokens: BigNumber;
+  token: UserToken;
+}
+
+const TokensTab = ({
+  activeTokens,
+  inactiveTokens,
+  totalTokens,
+  lockedTokens,
+  token,
+}: TokensTabProps) => {
+  const tokenDecimals = useMemo(
+    () => getTokenDecimalsWithFallback(token?.decimals),
+    [token],
+  );
+
+  const formattedTotalAmount = formatTokenValue({
+    value: totalTokens,
+    suffix: ` ${token?.symbol}`,
+    unit: tokenDecimals,
+    truncate: 3,
+  }).split(' ')[0];
+
+  return (
+    <>
+      <div className={styles.totalTokensContainer}>
+        <TokenIcon token={token || {}} size="xs" />
+        <p className={styles.totalTokens}>
+          {formattedTotalAmount} <span>{token?.symbol}</span>
+        </p>
+      </div>
```

Comment:
Large amounts break the design.

![Screenshot from 2021-03-01 17-43-29](https://user-images.githubusercontent.com/1193222/109521336-dec42d80-7ab5-11eb-8ce2-6cf99249c8c8.png)

I think that if you change the popover to have a min-width, it will resize accordingly

Reply:
(note that the 1px vertical line is actually the token icon being squished)